chain/consensus/filcns/upgrades.go

```diff
@@ -2609,43 +2609,56 @@ func PreUpgradeActorsV15(ctx context.Context, sm *stmgr.StateManager, cache stmg
 		workerCount /= 2
 	}
 
-	lbts, lbRoot, err := stmgr.GetLookbackTipSetForRound(ctx, sm, ts, epoch)
+	_, lbRoot, err := stmgr.GetLookbackTipSetForRound(ctx, sm, ts, epoch)
 	if err != nil {
 		return xerrors.Errorf("error getting lookback ts for premigration: %w", err)
 	}
 
 	config := migration.Config{
-		MaxWorkers:        uint(workerCount),
-		ProgressLogPeriod: time.Minute * 5,
+		MaxWorkers:              uint(workerCount),
+		ProgressLogPeriod:       time.Minute * 5,
+		PowerRampStartEpoch:     int64(buildconstants.UpgradeTuktukHeight),
```

Comment:
Since you're adding this to the general migration config please leave an issue in go-state-types for removing these fields after nv24.

Reply:
code comments will also help

Reply:
removed from config struct, added to method params but also added comments